README.md

```diff
@@ -75,6 +76,11 @@ or simply
 python -m ca_code.scripts.run_train <config.yml>
 ```
 
+(URHand training) You may need to unwrap images to get `color_mean.png` for static hand assets before launch training:
```

Comment:
Could you clarify when people might or might not need to run this?

Reply:
Also, is there an example config you could point to?

Reply:
Point to the config file.
I think people always need to run this as the released asset of hand do not contain color_mean so far.

Reply:
Makes sense! Could you add a note indicating that?

Reply:
Updated